core/templates/dev/head/pages/exploration-editor-page/services/learner-answer-details-data.service.spec.ts

```diff
@@ -86,19 +75,63 @@ describe('Learner answer details service', function() {
     $httpBackend.verifyNoOutstandingRequest();
   });
 
-  it('should successfully fetch learner answer info data from the backend',
+  describe(
+    'should successfully fetch learner answer info data from the backend',
```

Comment:
describe level descriptions should start with `when...`

Reply:
Done